samples/SQL/audit-sql-database-zone-redundant/azurepolicy.json

```diff
@@ -0,0 +1,33 @@
+{
+   "properties": {
+     "displayName": "Audit SQL Database is Zone Redundant",     
+     "description": "Will check is the SQL databases are configured for Zone Redundant",
+     "mode": "All",
+     "metadata": {
+       "category": "SQL"      
+     },
+     "parameters": {      
+      }
+     },
+     "policyRule": {
+       "if": {
+        "allOf": [
+          {
+            "field": "type",
+            "equals": "Microsoft.SQL/servers/databases"
+          },
+          {
+            "field": "name",
+            "notEquals": "master"
+          },
+          {
+            "field": "Microsoft.Sql/servers/databases/zoneRedundant",
+            "equals": "false"
```

Comment:
what is the default for this field?  If it is false then you also want to audit when the field does not exist (or audit when it is "notEquals": "true")

Reply:
great call ! the default is "zoneRedundant": false , I'll change to inequality, thanks!